### ISSUE TITLE
Font in top menu not elegant

### ISSUE THREAD

**Issue by martinheidegger** [consento-org/consento-website#32]
Currently the font in the top menu looks like this:

<img width="437" alt="Screen Shot 2020-02-05 at 14 19 44" src="https://user-images.githubusercontent.com/914122/73813207-a0146300-4822-11ea-8b2f-3f11d72886f2.png">

However, in previous designs that did not have a _brutal_ font, no uppercase letters, a smaller size and better spacing: https://github.com/consento-org/consento-website/issues/11#issuecomment-581209409

**Comment by Torgian** [comment]
My mistake, I overlooked this even though it's in the new design. Note: WorkSans-Medium, 21px, 27px line height, rgba(70,70,70,0.87)

**Comment by Torgian** [comment]
@martinheidegger for these, do you want them to turn red on hover, or change opacity? For now I will switch to changing opacity, but it can be reversed to turning red if you want.

**Comment by Torgian** [comment]
Screenshots below, please close / comment as necessary. [Commit Link](https://github.com/consento-org/consento-website/commit/19622aec98b8eb69d1eb00bb16e22bd2676e8500)

I had to add media queries for the nav bar so that nothing went off screen or overlapped.  First screen is from tablet mode.
<img width="385" alt="Screen Shot 2020-02-05 at 7 43 19 PM" src="https://user-images.githubusercontent.com/8596793/73839530-2e9fd900-4851-11ea-95a1-67641a428e3c.png">
<img width="1225" alt="Screen Shot 2020-02-05 at 7 42 56 PM" src="https://user-images.githubusercontent.com/8596793/73839536-32cbf680-4851-11ea-9360-d078d075786d.png">

**Comment by martinheidegger** [comment]
The spacing looked weird to me so I measured it in the browser and compared it to the design:
<hr/>
<img width="200" alt="Screen Shot 2020-02-05 at 23 02 28" src="https://user-images.githubusercontent.com/914122/73848799-1f7a5480-486c-11ea-90e5-90cc0d9555da.png">
<hr/>
→ Then I looked in the css spec and found a specified letter-spacing:
<hr/>
<img width="500" alt="Screen Shot 2020-02-05 at 23 04 45" src="https://user-images.githubusercontent.com/914122/73848916-5cdee200-486c-11ea-9a56-2ecc5df5edad.png">
<hr/>

→ Changing it to `0` was much closer to the actual design:

<hr/>
<img width="300" alt="Screen Shot 2020-02-05 at 23 09 09" src="https://user-images.githubusercontent.com/914122/73849026-8992f980-486c-11ea-848c-64676057f5ae.png">
<hr/>
→ I also noticed some vertical alignment differences:
<hr/>
<img width="500" alt="Screen Shot 2020-02-05 at 23 19 48" src="https://user-images.githubusercontent.com/914122/73849990-3f127c80-486e-11ea-821a-d0d676b1387e.png">
<img width="200" alt="Screen Shot 2020-02-05 at 23 20 13" src="https://user-images.githubusercontent.com/914122/73849994-420d6d00-486e-11ea-95e3-60c6eb16d572.png">
<hr/>
→ Which is probably a little harder to get right, but somehow the balance seems off. Maybe adjust the padding a little?
<hr/>

Furthermore I wondered if the colors are off. The screenshot was a bit blurry and the background darker, but after investigation...

<hr/>
<img width="133" alt="Screen Shot 2020-02-05 at 23 29 19" src="https://user-images.githubusercontent.com/914122/73850709-8e0ce180-486f-11ea-83b5-c55223b5ad08.png">
<hr/>

→ ... it turns out the colors are off indeed. The design had `#000` while the web color is `#3D444A`. So I looked into the CSS again and it turns out that you did set the color correctly:

https://github.com/consento-org/consento-website/blob/19622aec98b8eb69d1eb00bb16e22bd2676e8500/partials/navigation.hbs#L11

but the menu.sass file defines in a stronger selector _(`.menu-item a` is weaker than `#menu ul a`)_ the color as `inherit`:

https://github.com/consento-org/consento-website/blob/19622aec98b8eb69d1eb00bb16e22bd2676e8500/assets/main/sass/layout/_menu.scss#L23

....which means that the color of the menu list is used instead of the color of the list item:

https://github.com/consento-org/consento-website/blob/19622aec98b8eb69d1eb00bb16e22bd2676e8500/assets/main/sass/layout/_menu.scss#L12

... it looks like #42 is the issue that should be solved first: Move all the design back into the sass file and generate one common CSS file that is used. It might even be helping with #43.

<hr/>

Also in regards to the colors: The button color is blue (`#008BDC`) in @marcchataigner's design and not `consento-blue`. Which is probably on purpose :wink:.

<hr/>

One last thing to mention about the button, in the design's vertical/horizontal ratio is closer to rectangular and vertically centered:

<img width="400" alt="Screen Shot 2020-02-05 at 23 53 51" src="https://user-images.githubusercontent.com/914122/73852605-ccf06680-4872-11ea-8579-903d8980ab1d.png">

This is a common webdesign problem that can be solved either the hard way, by adjusting every vertical layout by hand - which can be impossible for some devices as the font-rendering is different, or it can be solved [using flexbox](https://www.smashingmagazine.com/2013/08/absolute-horizontal-vertical-centering-css/)

**Comment by Torgian** [comment]
I am going to review this right now. Since we made some changes yesterday with the css migration, it may have affected this issue.

**Comment by Torgian** [comment]
It appears the design problems are still there, so I will start working on this.

**Comment by Torgian** [comment]
Please refer to PR [61](https://github.com/consento-org/consento-website/pull/61)

I made a couple of minor changes. This seems to have fixed _some_ spacing issues.

**Comment by Torgian** [comment]
<img width="1254" alt="Screen Shot 2020-02-16 at 5 40 07 PM" src="https://user-images.githubusercontent.com/8596793/74602440-6aedf780-50e3-11ea-9c5e-e8dbe61502e7.png">

**Comment by Torgian** [comment]
<img width="562" alt="Screen Shot 2020-02-16 at 5 41 57 PM" src="https://user-images.githubusercontent.com/8596793/74602469-b0aac000-50e3-11ea-891e-af5ba9285d19.png">

**Comment by martinheidegger** [comment]
> This seems to have fixed some spacing issues.

Looking forward to all issues fixed.

**Comment by Torgian** [comment]
@consento-org/design I need confirmation on the button colors. In Zeplin, the border colors are #2989d6, and the text is `rgba(9, 120, 208, 0.87)` ( or hex #0978D0 ) Is this correct, or do the colors need to be one or the other?

**Comment by marcchataigner-zz** [comment]
@Torgian on the zeplin file, I see that the blue is the `consent-blue` : #37afc9 for buttons (the `test your privacy level`, `be the first to use it`, etc. Is it the color you are looking for?

**Comment by Torgian** [comment]
Oh, my mistake. I meant the Download button in the navigation menu.

**Comment by marcchataigner-zz** [comment]
Ah, I see. The way it runs now in the `write.georepublic.net`, it looks good to me. The blue edge is slightly darker than the filled in blue shade when the mouse over comes, and it is the correct dynamic for me.  :-)

**Comment by marcchataigner-zz** [comment]
Only maybe that I look at it, the `download` label could be aligned in height with the button center, like you have in other buttons. Thee text can be 1 or 2 px down.   But this is not a key requirement, just optical illusion fun   :-D 

![Screen Shot 2020-02-17 at 11 33 45](https://user-images.githubusercontent.com/3974900/74619577-62480080-5179-11ea-8da2-b0a74a2fd0a9.png)

**Comment by Torgian** [comment]
Yes, I'm working on this right now actually. I started by removing the download_btn class and adding things one by one. When I remove it, everything is centered:

<img width="345" alt="Screen Shot 2020-02-17 at 10 32 10 AM" src="https://user-images.githubusercontent.com/8596793/74619672-4fc9c900-5171-11ea-98ec-9d555480706c.png">

I narrowed it down to the culprit: It's the font. If I remove all the styling, and change the font to Catamaran-SemiBold, it looks off-center ( by exactly one pixel. ) It's annoying.

I've tried a few simple fixes, but it ends up changing the overall look of the button. 

No styling ( other than default styling from ghost / bootstrap ): 20 px top and bottom.

<img width="345" alt="Screen Shot 2020-02-17 at 10 32 10 AM" src="https://user-images.githubusercontent.com/8596793/74619980-6886ae80-5172-11ea-96d5-ace20469d46a.png">

Change the font : 

<img width="114" alt="Screen Shot 2020-02-17 at 10 38 48 AM" src="https://user-images.githubusercontent.com/8596793/74619988-72101680-5172-11ea-925e-df908db755d3.png">

Keep in mind that bootstrap automatically aligns text in buttons to the middle ( it's part of the `.btn` class )

**Comment by Torgian** [comment]
Fixed. Refer to this [commit](https://github.com/consento-org/consento-website/commit/77a065b8ece6786a68a4d6cbdd92bf5538687c22) While the font family changes how the text is spaced, the font size also plays a part. I found that it was being over-ridden by ghost's base styling, so I added the !important tag and reduced the font size by one. This gives it a centered appearance. 

<img width="175" alt="Screen Shot 2020-02-17 at 11 00 10 AM" src="https://user-images.githubusercontent.com/8596793/74620729-0a0eff80-5175-11ea-8767-4b9ee29822b0.png">


<img width="179" alt="Screen Shot 2020-02-17 at 11 00 07 AM" src="https://user-images.githubusercontent.com/8596793/74620726-08453c00-5175-11ea-8687-a28ede3aacb5.png">
